tlf/handle_test.go

```diff
@@ -241,6 +285,30 @@ func TestHandleAccessorsPublic(t *testing.T) {
 		})
 }
 
+func TestHandleAccessorsSingleTeam(t *testing.T) {
+	w := []keybase1.UserOrTeamID{
+		keybase1.MakeTestTeamID(4).AsUserOrTeam(),
+	}
+
+	h, err := MakeHandle(
+		w, nil, nil, nil, nil)
+	require.NoError(t, err)
+
+	require.Equal(t, SingleTeam, h.Type())
+
+	// All types of IsWriter/IsReader calls should panic for team TLFs.
```

Comment:
Yeah, I thought about that.  Not really sure how to consolidate them.  There are three `IsTeamWriter()` calls in libkbfs right now that all do slightly different things: `bare_root_metadata_v3.go` (checks that the actual _device_ is keyed, not just that the user is a writer), `root_metadata.go` (pretty standard check using a `libkbfs.TlfHandle`) and `mdserver_local_shared.go` (checks for a writer OR a rekeyer, using a `tlf.Handle`).

Once the server is ready to do these checks, we can think about factoring one of these out into a nice function, but I'm not sure which one is best yet. But if you have a good idea how to improve it now, I'm open to it.